app/models/opening_time.rb

```diff
@@ -0,0 +1,57 @@
+class OpeningTime < ApplicationRecord
```

Comment:
The `open_s*` attributes are there because we need something to specify if we should be validating opening times. For each opening time we have an extra attribute ex: `open_saturday`. I think we could rename these attributes to something like `will_open_saturday` and keep the instance methods `open_saturday?` and `open_sunday?` Do you think that's better?

Reply:
Ahhh, I see. I misunderstood. One is not necessarily implied by the other. In that case I don't think a rename is useful either (i.e. `open_saturday?` already conveys enough meaning from my perspective